vulture/core.py

```diff
@@ -176,8 +181,9 @@ def scavenge(self, paths):
 
         for module in included_modules:
             self.log('Scanning:', module)
+            module_string = _get_stdlib_whitelist()
             try:
-                module_string = read_file(module)
+                module_string += read_file(module)
```

Comment:
I think it's better to use pkgutil.get_data. The change is more complicated than anticipated. Let me propose the following solution:

```
    def scavenge(self, paths):
        def exclude(name):
            return any(fnmatchcase(name, pattern) for pattern in self.exclude)

        for module in self._get_modules(paths):
            if exclude(module):
                self.log('Excluded:', module)
                continue

            self.log('Scanning:', module)
            try:
                module_string = read_file(module)
            except VultureInputException as err:
                print('Error: Could not read file %s - %s' % (module, err))
                print('You might want to change the encoding to UTF-8.')
            else:
                self.scan(module_string, filename=module)

        whitelist_names = ['stdlib.py']
        for name in whitelist_names:
            path = os.path.join('whitelists', name)
            if exclude(path):
                self.log('Excluded whitelist:', path)
            else:
                module_data = pkgutil.get_data('vulture', path)
                if module_data is None:
                    sys.exit('Error: Please use "python -m vulture.core".')
                module_string = module_data.decode("utf-8")
                self.scan(module_string, filename=path)
```

There are more changes needed afterwards: the tests have to use `python -m vulture.core` instead of `python vulture.py`.